packages/firebase_crashlytics/firebase_crashlytics/CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## 0.2.0-dev.1
+
+* **Breaking change**: Migration to the new Firebase Crashlytics SDK. Follow the [migration guide](https://firebase.google.com/docs/crashlytics/upgrade-sdk?platform=android) (steps 1 and 2) to update your app.
+* **Breaking change**: the following methods have been removed:
+  * `setUserEmail`
+  * `setUserName`
+  * `getVersion`
+  * `isDebuggable`
+* **Breaking change**: the methods `setInt`, `setDouble`, `setString` and `setBool` have been replaced by `setCustomKey`.
+* Logs and keys are sent to Crashlytics as they're set.
+* Fixes a bug that prevented keys from being set on iOS devices.
```

Comment:
Should the changelog also add mention of bumping the Gradle plugin used? I've seen this documented in the changelogs for other first-party plugins before

Reply:
Covered by the migration guide. Added a link in the changelog to point to the migration guide.